xmake/core/main.lua

```diff
@@ -74,6 +85,11 @@ function main._help()
             utils.cprint(menu.copyright)
         end
 
+        -- print features
+        if menu.features then
```

Comment:
I think it's not necessary to show the feature list in help menu.  It seems that users don't need to care about this
 😸